src/main/java/tasklist/Task.java

```diff
@@ -0,0 +1,25 @@
+package tasklist;
+
+public abstract class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone(){
+        isDone = true;
+        getStatusIcon();
```

Comment:
Happen to think, can try removing this line, should be not required anymore

Reply:
Thanks for the advice, will improvise the code accordingly!